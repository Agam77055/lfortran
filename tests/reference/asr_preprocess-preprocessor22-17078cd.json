{
    "basename": "asr_preprocess-preprocessor22-17078cd",
    "cmd": "lfortran --cpp --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/preprocessor22.f90",
    "infile_hash": "89705ba1894edbbf22451a006c8b1520aaad77f9f3d6da1027acdca4",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "asr_preprocess-preprocessor22-17078cd.stdout",
    "stdout_hash": "0affd15f0b7b5776844c076bbecfd400b6c6fe328785ff2983b5e9fe",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}