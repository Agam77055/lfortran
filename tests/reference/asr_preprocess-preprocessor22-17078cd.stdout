(TranslationUnit
    (SymbolTable
        1
        {
            preprocessor22:
                (Program
                    (SymbolTable
                        2
                        {
                            
                        })
                    preprocessor22
                    []
                    [(Print
                        (StringConstant
                            "Both VAR1 and VAR2 are defined."
                            (String 1 (IntegerConstant 31 (Integer 4) Decimal) .false. .false. PointerString)
                        )
                    )
                    (Print
                        (StringConstant
                            "At least one of VAR3 or VAR4 is defined."
                            (String 1 (IntegerConstant 40 (Integer 4) Decimal) .false. .false. PointerString)
                        )
                    )]
                )
        })
    []
)
